docs/user/private-app-repository.md

```diff
@@ -0,0 +1,105 @@
+- `persistence.enabled`: We will set this value to `true` to enable persistence for the charts we store. Note that this will create a [Kubernetes Persistent Volume Claim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#lifecycle-of-a-volume-and-claim) so depending on your Kubernetes provider you may need to manually allocate the required Persistent Volume to satisfy the claim. Some Kubernetes providers will automatically create PVs for you so setting this value to `true` will be enough.
+
+<img src="../img/chartmuseum-deploy-form.png" alt="ChartMuseum Deploy Form" width="600px">
+
+### ChartMuseum: Upload a Chart
+
+Once ChartMuseum is deployed you will be able to upload a chart. In one terminal open a port-forward tunnel to the application:
+
+```console
+$ export POD_NAME=$(kubectl get pods --namespace default -l "app=chartmuseum" -l "release=my-chartrepo" -o jsonpath="{.items[0].metadata.name}")
+$ kubectl port-forward $POD_NAME 8080:8080 --namespace default
+Forwarding from 127.0.0.1:8080 -> 8080
+Forwarding from [::1]:8080 -> 8080
+```
+
+And in a different terminal you can push your chart:
+
+```console
+$ helm package /path/to/my/chart
+Successfully packaged chart and saved it to: /path/to/my/chart/my-chart-1.0.0.tgz
+$ curl --data-binary "@my-chart-1.0.0.tgz" http://localhost:8080/api/charts
+{"saved":true}
+```
+
+### ChartMuseum: Configure the repository in Kubeapps
+
+To add your private repository go to `Configuration > App Repositories` in Kubeapps and click on "Add App Repository". You will need to add your repository using the Kubernetes DNS name for the ChartMuseum service. This will be `<release_name>-chartmuseum.<namespace>:8080`:
+
+<img src="../img/chartmuseum-repository.png" alt="ChartMuseum App Repository" width="600px">
+
+Once you create the repository you can click on the link for the specific repository and you will be able to deploy your own applications using Kubeapps.
+
+### ChartMuseum: Authentication/Authorization
+
+It is possible to configure ChartMuseum to use authentication with two different mechanism:
+
+- Using HTTP [basic authentication](https://chartmuseum.com/docs/#basic-auth) (user/password). To use this feature, it's needed to:
+  - Specify the parameters `secret.AUTH_USER` and `secret.AUTH_PASS` when deploying the ChartMuseum.
+  - Set as Authorization Header a `Basic` auth header using as value the base64 codification of the string `user:password`. For example, for the user "Aladdin" and password "open sesame", it would use the following header field: `Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==`
+- Using a [JWT token](https://github.com/chartmuseum/auth-server-example). Once you obtain a valid token you can set it in the Authorization Header field of the App Repository form. Note that in this case it will be prefixed with `Bearer` . For example: `Bearer UVd4aFpHUnBianB2Y0dWdUlIT=`.
```

Comment:
extra space
```suggestion
- Using a [JWT token](https://github.com/chartmuseum/auth-server-example). Once you obtain a valid token you can set it in the Authorization Header field of the App Repository form. Note that in this case it will be prefixed with `Bearer`. For example: `Bearer UVd4aFpHUnBianB2Y0dWdUlIT=`.
```